realm/realm-library/src/androidTest/java/io/realm/OrderedRealmCollectionTests.java

```diff
@@ -221,6 +221,14 @@ public void first_emptyCollection() {
     }
 
     @Test
+    public void first_withDefault() {
+        AllJavaTypes defaultObject = collection.get(0);
```

Comment:
I thought to do that at first, but I wasn't sure how to determine and instantiate the class contained in the collection within the test.